onnxruntime/core/graph/model.cc

```diff
@@ -202,6 +220,23 @@ Model::Model(ModelProto&& model_proto, const PathString& model_path,
   // create instance. need to call private ctor so can't use make_unique
   GSL_SUPPRESS(r .11)
   graph_.reset(new Graph(*this, model_proto_.mutable_graph(), domain_to_version, IrVersion(), schema_registry, model_local_functions, logger));
+
+  std::vector<FunctionTemplate*> model_local_func_templates;
```

Comment:
Is it possible to factor out this code and the code above (which look like they are the same/similar) into one common function?